install/k8s/install.sh

```diff
@@ -235,22 +235,22 @@ else
 fi
 
 echo "Applying contiv installation"
-echo "$netmaster netmaster" >> /etc/hosts
+sudo grep -q -F "netmaster" /etc/hosts || sudo echo "$netmaster netmaster" >> /etc/hosts
```

Comment:
`sudo echo` will not work because the file handle is opened before `sudo` is executed

`sudo` is also not required to read /etc/hosts

Try this:  `grep -q -F "netmaster" /etc/hosts || echo "$netmaster netmaster" | sudo tee -a /etc/hosts`